docs/changes.rst

```diff
@@ -19,6 +19,29 @@ Released: not yet
 * Installation of this package using "setup.py" is no longer supported.
   Use "pip" instead.
 
+* Update to pywbem version 1.7.0 may cause issues with SSL because Python
+  packages may now require support of TLS version GE 1.2. Because the the
```

Comment:
I suggest to use ">=" instead of "GE".

Duplicate: "the the"

Reply:
DONE